app/controllers/application_controller.rb

```diff
@@ -29,6 +29,7 @@ class ApplicationController < ActionController::Base
   prepend_before_action :add_new_relic_trace_attributes
   prepend_before_action :session_expires_at
   prepend_before_action :set_locale
+  prepend_before_action :browser_is_ie11?
```

Comment:
Where is this defined? Is it referencing the method in `SessionsController`? If so, what's the reason it needs to be called before every action in the application controller?

Reply:
oh whoops i need to clean up